extensions/jira-search/src/types.ts

```diff
@@ -0,0 +1,2 @@
+export const issueFilters = <const>["allIssues", "issuesInOpenSprints", "myIssues", "myIssuesInOpenSprints"]
```

Comment:
As both of the types here are only used inside `issue.ts` I don't see a benefit of having these two in their own file and would move them to `issue.ts` instead.